packages/frontend/src/components/UserSettings/ProjectManagementPanel/index.tsx

```diff
@@ -73,8 +77,10 @@ const ProjectListItem: FC<ProjectListItemProps> = ({
                     <Can
                         I="delete"
                         this={subject('Project', {
+                            type: project.type,
+                            projectUuid: project.projectUuid,
                             organizationUuid: user.data?.organizationUuid,
-                            projectUuid,
+                            createdByUserUuid: project.createdByUserUuid,
```

Comment:
should this be 
`createdByUserUuid: user.data?.uuid` ? 
otherwise, the other check is redundant, right ?

Reply:
```ts
        can('delete', 'Project', {
            type: ProjectType.PREVIEW,
            createdByUserUuid: member.userUuid,
        });
```

this is the permission definition so we have to match `project.createdByUserUuid` to the member's uuid

Reply:
we have similar for `CsvJobResult`

```ts
        can('view', 'CsvJobResult', {
            createdByUserUuid: member.userUuid,
        });
```